song-server/src/main/java/org/icgc/dcc/song/server/repository/mapper/AnalysisMapper.java

```diff
@@ -39,12 +39,13 @@ public Analysis map(int index, ResultSet r, StatementContext ctx) throws SQLExce
     val study = r.getString("study_id");
     val type = r.getString("type");
     val state = r.getString("state");
+    val info = r.getString("info");
```

Comment:
We've been putting all the table field names directly in the repository mapper classes up until now; is there any benefit from putting the field names into constants? If we do it, we should do it everywhere...